improver/nbhood/recursive_filter.py

```diff
@@ -107,27 +114,30 @@ def __init__(self, alpha_x=None, alpha_y=None, iterations=None,
                 warnings.warn(
                     "More than two iterations degrades the conservation"
                     "of probability assumption.")
-        self.alpha_x = alpha_x
-        self.alpha_y = alpha_y
+        self.smoothing_coefficient_x = smoothing_coefficient_x
+        self.smoothing_coefficient_y = smoothing_coefficient_y
         self.iterations = iterations
         self.edge_width = edge_width
         self.re_mask = re_mask
 
     def __repr__(self):
         """Represent the configured plugin instance as a string."""
-        result = ('<RecursiveFilter: alpha_x: {}, alpha_y: {}, iterations: {},'
-                  ' edge_width: {}')
-        return result.format(self.alpha_x, self.alpha_y, self.iterations,
-                             self.edge_width)
+        result = ('<RecursiveFilter: smoothing_coefficient_x: {}, '
+                  'smoothing_coefficient_y: {}, iterations: {}, '
+                  'edge_width: {}')
+        return result.format(
+            self.smoothing_coefficient_x, self.smoothing_coefficient_y,
+            self.iterations, self.edge_width)
 
     @staticmethod
-    def _recurse_forward(grid, alphas, axis):
+    def _recurse_forward(grid, smoothing_coefficients, axis):
         """
         Method to run the recursive filter in the forward direction.
 
         In the forward direction:
             Recursive filtering is calculated as:
-                Bi = ((1-alpha) * Ai) + (alpha * Bi-1)
+                Bi = ((1-smoothing_coefficient) * Ai) +
```

Comment:
The formatting of this and the bit below could be tidied up a bit, it's not the clearest in the sphinx documentation

Reply:
And I think the same docstring is used below

Reply:
I've made it sphinx happy, which unfortunately means it is less readable as a doc-string. C'est la vie.